docs/source/release/v3.14.0/reflectometry.rst

```diff
@@ -81,6 +81,9 @@ Improved
 ########
 
 - :ref:`algm-ReflectometryReductionOneAuto` now supports the Wildes method for polarization corrections as well as Fredrikze when configured in the parameters file.
+- :ref:`algm-ReflectometryReductionOneAuto` now uses spectrum numbers for it's processing instructions instead of workspace indecies
+- :ref:`algm-ReflectometryReductionOne`, :ref:`algm-ReflectometryReductionOneAuto`, :ref:`algm-CreateTransmissionWorkspace` and :ref:`algm-CreateTransmissionWorkspaceAuto` now uses spectrum numbers for their processing instructions instead of workspace indecies
```

Comment:
Minor spelling and grammer fixes:
- `now use` rather than `now uses`
- `indices` rather than `indecies`